vendor-extra/LiberoPageBundle/src/EventListener/SiteHeaderListener.php

```diff
@@ -37,6 +37,12 @@ public function onCreatePagePart(CreatePagePartEvent $event) : void
                             'alt' => $this->translate('libero.page.site_name', $context),
                         ],
                     ],
+                    'menu' => [
```

Comment:
This is clearly simplistic and not (easily) changeable, but the menu structure is likely to change in the near future.